src/wirecloud/platform/widget/utils.py

```diff
@@ -36,4 +81,229 @@ async def get_or_add_widget_from_catalogue(db: DBSession, vendor: Vendor, name:
         if await resource.is_available_for(db, user):
             return await get_widget_from_resource(db, resource.id), resource
 
-    return None
\ No newline at end of file
+    return None
+
+
+def xpath(tree: _ElementTree, query: str, xmlns: Any) -> Union[
+    list[_Element], list[str], str, float, bool, _Element, None]:
+    if xmlns is None:
+        query = query.replace('xhtml:', '')
+        return tree.xpath(query)
+    else:
+        return tree.xpath(query, namespaces={'xhtml': xmlns})
+
+
+_widget_platform_style = {}
+
+
+def get_widget_platform_style(request: Request, theme: str):  # TODO: add _widget_platform_style type
```

Comment:
Revísalo, pero según el código parece ser una tupla de str.
```suggestion
def get_widget_platform_style(request: Request, theme: str) -> tuple[str]:
```